aether_server/static/js/main.js

```diff
@@ -0,0 +1,63 @@
+let videoPlayer = document.querySelector("#videoplayer")
+let startButton = document.querySelector("#start")
+
+
+const socket = new WebSocket("http://localhost:7878/ws");
```

Comment:
Hardcoding endpoints is not good.

TODO: Refer to `jinja` or other Pythonic template-dependencies to update this during server-side rendering.